core/db/sql/migrations/00006_sla_rollups.sql

```diff
@@ -0,0 +1,21 @@
+-- +migrate Up
+create table sla_rollups(
+  id serial primary key,
```

Comment:
should we include a `tx_hash` field here like the KV txs have, or will that not be necessary?

Reply:
let's add it yeah, can you put a method on the CoreApplication struct to generate that easily?

should be something like
```
func (app *CoreApplication) toTxHash(tx []byte) string {
  return hex.EncodeToString(sha256.Sum256(tx)[:])
}
```